src/lib/Microsoft.Health.Events/Telemetry/Exceptions/EventHubConfigurationExceptionTelemetryProcessor.cs

```diff
@@ -0,0 +1,63 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using EnsureThat;
+using Microsoft.Health.Common.Telemetry;
+using Microsoft.Health.Logging.Telemetry;
+
+namespace Microsoft.Health.Events.Telemetry.Exceptions
+{
+    public class EventHubConfigurationExceptionTelemetryProcessor
```

Comment:
Note: @pallar-ms has an upcoming change that will better process handled vs. unhandled exceptions and allow us to derive this class (`EventHubConfigurationExceptionTelemetryProcessor`) from a base class.